src/components/ContactMe/ContactForm/styles.css

```diff
@@ -0,0 +1,44 @@
+.contact-form-content {
+  width: 100%;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  grid-gap: 24px;
+}
+
+.name-container {
+  display: flex;
+  align-items: center;
+  grid-gap: 24px;
+}
+
+form input,
+form textarea {
+  flex: 1;
+  width: 100%;
+  font-size: 15px;
+  color: #fff;
+  background: #130f2a;
+  border-radius: 9px;
+  border: 1.5px solid #6751b9;
+  padding: 16px;
+}
+
+form button {
+  font-size: 16px;
+  font-weight: 500;
+  color: #fff;
```

Comment:
Remove text parameters from here. just use TEXT component inside button element, it will automatically fix this part.